### PR TITLE
Downgrade to node.js v10.24.1 for ExtractFiles task

### PR DESCRIPTION
**Pull request by am11**
This is a workaround for https://github.com/microsoft/azure-pipelines-tasks/issues/15261.

cc @ViktorHofer

### CODE REVIEW

**Comment by am11** [comment]
cc @wfurt, @safern, learned it the hard way that _this_ was the reason to keep node.js to v10 rather than the latest LTS .. 😁

**Review by mthalman — APPROVED** [review]
Thanks for documenting in the Dockerfile why this pinned version is needed.

**Comment by am11** [comment]
Good to merge? I'll update runtime PR once this is merged. :)

**Review by mthalman — APPROVED** [review]

**Pull request merged**
